packages/mui-babel-macros/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/internal-babel-macros",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Updated version due to https://github.com/mui/material-ui/pull/41425

Reply:
So we need to update the version? cc @michaldudak

Reply:
Yes, this will get published to npm as well.